R/crandb-public-api.R

```diff
@@ -36,11 +41,26 @@ cran_package <- function(name, version = NULL) {
 #' @param format What to return. \sQuote{`short`} means the
 #'    title and version number only. \sQuote{`latest`} means
 #'    the complete description of the latest version. \sQuote{`full`}
-#'    means all versions.
+#'    means all versions. Note that the output printing look the same 
+#'    for all formats, although the output is actually different.
```

Comment:
I think we should just return tibbles.

Reply:
Ok, can I leave it this way in this PR and then I'll do a PR tackling #43?

Reply:
ok.